src/coresyf_image_crop/tests/test_image_crop.py

```diff
@@ -29,5 +32,15 @@ def test_get_shapefile_polygon_extent(self):
         self.assertEqual(extent_polygon.ExportToWkt(), self.extent_polygon_wkt)
 
     def test_get_shapefile_crs(self):
-        espg_code = get_shapefile_crs(self.data_source)
-        self.assertEqual(espg_code, 3763)
\ No newline at end of file
+        epsg_code = get_shapefile_crs(self.data_source)
+        self.assertEqual(epsg_code, self.epsg_code)
+
+    def test_crop_raster(self):
+        test_image = 'test_data/Aveiro_resampled.tif'
+        output_image = os.path.join(TEMP_PATH, 'output')
```

Comment:
You can move this and previous line to the "setup" method.

Reply:
done